src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -18,7 +19,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
 {
     internal interface IDeclaredSymbolInfoFactoryService : ILanguageService
     {
-        bool TryGetDeclaredSymbolInfo(StringTable stringTable, SyntaxNode node, out DeclaredSymbolInfo declaredSymbolInfo);
+        bool TryGetDeclaredSymbolInfo(StringTable stringTable, SyntaxNode node, string rootNamespace, out DeclaredSymbolInfo declaredSymbolInfo);
+
+        bool TryGetTargetTypeName(SyntaxNode node, out string instanceTypeName);
+
+        string GetRootNamespace(CompilationOptions compilationOptions);
```

Comment:
why is this here?

Reply:
because it's convenient :)
I couldn't find an exisit helper for this. Any suggestions where I should put this?